pkgs/development/python-modules/pypi-sources.json

```diff
@@ -0,0 +1,80 @@
+{
+  "toolz": {
+    "latest_version": "0.7.4",
+    "meta": {
```

Comment:
thanks @garbas 
As far as I can see the JSON API doesn't give such meta data for each version. It can definitely change in setup.py (license, or trove classifier) but I am not using that data.